packages/lodestar/src/network/util.ts

```diff
@@ -8,7 +8,9 @@ import PeerInfo from "peer-info";
 import promisify from "promisify-es6";
 import {Shard} from "@chainsafe/eth2.0-types";
 
-import {RequestId, SHARD_SUBNET_COUNT, SHARD_ATTESTATION_TOPIC, BLOCK_TOPIC, ATTESTATION_TOPIC} from "../constants";
+import {ATTESTATION_TOPIC, BLOCK_TOPIC, RequestId, SHARD_ATTESTATION_TOPIC, SHARD_SUBNET_COUNT} from "../constants";
+import fs from "fs";
+import {loadPeerId} from "./nodejs";
```

Comment:
I was trying to keep the nodejs specific code in ./nodejs
Now this is importing the fs module into a non-nodejs-specific module